src/main/java/frc/robot/commands/ClimbDown.java

```diff
@@ -25,7 +25,7 @@ public ClimbDown() {
     addRequirements(this.climber);
   }
 
-  // Called when the command is initially scheduled.
+  /** Called when the command is initially scheduled. */
```

Comment:
@Jack-Haefele this should absolutely definitely not be a JavaDoc comment, since we don't want it to override the JavaDoc from the superclass

Reply:
whoops